.github/workflows/pr-title.yml

```diff
@@ -27,6 +27,7 @@ jobs:
             ci
             chore
             revert
+            security
```

Comment:
For my own education - was this not already a thing?

Reply:
Ah, I see - not for this repo

Reply:
Since we are using conventional commits how it was intended, Im not sure a security type makes sense here. Normally you would put a security time as a fix, so it would cut a new release

Reply:
Was trying to make this consistent across our repos. I can revert it though.